README.md

```diff
@@ -210,7 +209,7 @@ make test-unit
 ```
 
 For finer control over which tests get executed, e.g. to run all tests in a specific file, activate
-the [virtualenv](#virtualenv) and run:
+the [virtualenv](Makefile) and run:
```

Comment:
This doesn't really have anything to do with activating the venv... that would be `./venv/bin/activate`

Reply:
Fixed. (It was just a bad link to virtualenv section that does not exist; it should be virtual-environment.)